tools/local-nodescanner/main.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"os"
+	"path/filepath"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/stackrox/scanner/pkg/analyzer/nodes"
+)
+
+func main() {
+	flag.CommandLine = flag.NewFlagSet(os.Args[0], flag.ExitOnError)
+	fPath := flag.String("fspath", "/host", "Path to the root folder of a filesystem")
+	fRHCOSrequired := flag.Bool("rhcosRequired", true, "Fails scan if path does not contain a filesystem generated by RHCOS. Node Scanning only works on RHCOS.")
+	fUncertifiedRHEL := flag.Bool("uncertifiedRhel", false, "Set true for CentOS and false for RHEL.")
```

Comment:
nit: `uncertifiedRHEL` (not sure if that breaks flag name conventions)

Reply:
That would make it harder to use in the command line, assuming it's one of the intended use-cases of the tool? I would use `rhcos-required` or `rchosrequired`, honestly.

Reply:
I have renamed both flags, see https://github.com/stackrox/scanner/pull/1164/commits/07e33824d9936c620f60d2a017297ec5a06fd3cc